CONTRIBUTING.md

```diff
@@ -8,6 +8,9 @@ See [Superchain Upgrades] OP-Stack specifications.
 
 ## Adding a chain
 
+### 0. Install dependencies
+You will need [`jq`](https://jqlang.github.io/jq/download/) and [`foundry`](https://book.getfoundry.sh/getting-started/installation) installed.
```

Comment:
Do we have a sense of how "bad" it is to require foundry, here? We are already requiring Go, so if the script were transcribed to Go we wouldn't need to add the dependency.

Reply:
Currently you need foundry to deploy contracts for a chain anyway, so I think this is ok for now